Assets/MRTK/SDK/Features/UX/Scripts/Collections/GridObjectCollection.cs

```diff
@@ -333,38 +362,70 @@ protected void ResolveGridLayout(Vector3[] grid, LayoutOrder order)
             {
                 startOffsetY = yMax * CellHeight;
             }
+            float alignmentOffsetX = 0;
+            float alignmentOffsetY = 0;
 
             if (layout == LayoutOrder.ColumnThenRow)
             {
                 for (int y = 0; y < yMax; y++)
+                {
                     for (int x = 0; x < xMax; x++)
                     {
+                        if (y == yMax - 1)
```

Comment:
I think that this code means that the first row or column will never get affected by this (i.e. it will always lay out with a left or top most alignment) - this could be fine, but it might be worth doc-ing that as well (i.e. this only intentionally affect the second row or column on and on)

Or maybe it should affect the first as well - I think the main feedback here is the intent should be called out intentionally so that there's no confusion in the future if it's a bug or a feature.

Reply:
Because the grid is filled from left to right and then top to bottom, the alignment behavior will only ever be visible on the last row/column. I think we'd need to address a different set of needs if we wanted to let users define a different "order" or how objects should be laid out. As far as the scope of addressing the initial issue is concerned, I think this covers what they were looking for.